### ISSUE TITLE
Set cwd when running programs

### ISSUE THREAD

**Issue by simonlindholm** [Kattis/problemtools#142]
Currently programs are run with the problem directory as cwd; this doesn't match e.g. Kattis.

**Comment by austrin** [comment]
It's not clear to me what the right choice is here.

The problem format standard does not specify any guarantee on the cwd of executed program and we probably agree that it pretty clearly shouldn't, right?

Of course problemtools could make a different arbitrary choice than the current one.

**Comment by simonlindholm** [comment]
Yeah, we noticed in a recent contest (https://google19.kattis.com/problems/google19.dictionary) that Kattis's setup is run programs with cwd equal to an empty directory `/run`, and submitted files accessible from `/src`, and had to make a clarification about that.

I would definitely prefer if the standard specified directory setup and the cwd, but you're right that there appears to be no clear right choice.

**Comment by gkreitz** [comment]
The 2023-07-draft standard now specifies a working directory, https://www.kattis.com/problem-package-format/spec/2023-07-draft.html#working-directory . I think it would make sense to just follow the same spec for legacy.

We do need to implement the behavior on the Kattis side too.

**Comment by gkreitz** [comment]
Problemtools now does this. Kattis does not (yet).
